src/types/quizzes/interfaces/quizzes.interface.ts

```diff
@@ -11,3 +17,12 @@ export interface Quiz extends CommonEntityFields {
   author: Pick<UserResponse, '_id'>
   category: Category | null
 }
+
+export interface QuizViewProps {
+  questions: Question[]
+}
+
+export interface QuizQuestionProps {
```

Comment:
we always define props interface in the component file

Reply:
> we always define props interface in the component file

moved it back to the component